nilearn/decoding/decoder.py

```diff
@@ -0,0 +1,849 @@
+"""High-level decoding object that exposes standard classification and
+regression strategies such as SVM, LogisticRegression and Ridge, with optional
+feature selection, and integrated hyper-parameter selection.
+"""
+# Authors : Yannick Schwartz
+#           Andres Hoyos-Idrobo
+#
+# License: simplified BSD
+
+import itertools
+import warnings
+from distutils.version import LooseVersion
+
+import numpy as np
+import sklearn
+from sklearn import clone
+from sklearn.base import RegressorMixin
+from sklearn.externals.joblib import Parallel, delayed
```

Comment:
```suggestion
try:
    from sklearn.externals.joblib import Parallel, delayed
except ImportError:
    from joblib import Parallel, delayed
```
Scikit-learn will soon stop vendoring joblib